docs/styleguide/PyGuide.md

```diff
@@ -236,6 +237,31 @@ with open("hello.txt") as hello_file:
         print(line)
 ```
 
+
+<a id="s2.9-abstract-classes"></a>
+<a id="29-abstract-classes"></a>
+<a id="abstract-classes"></a>
+### 2.9 Abstract Classes 
+
+When defining user abstract classes, the following template should be used:
+
+```python
+from abc import ABC, abstractmethod
```

Comment:
Are you allowing an abstract property? Could you add this to the template?

Reply:
Done.